lib/src/interface/local_platform.dart

```diff
@@ -48,5 +48,5 @@ class LocalPlatform extends Platform {
   String get version => io.Platform.version;
 
   @override
-  bool get ansiSupported => io.Platform.ansiSupported;
+  bool get ansiSupported => io.stdin.supportsAnsiEscapes;
```

Comment:
There is also now `stdout.supportsAnsiEscapes`. We currently don't need it, but maybe we should make sure it can be added easily in a non-confusing way? Would it make sense to call this `stdinAnsiSupported`?